src/renderer/main.js

```diff
@@ -13,6 +13,8 @@ import { register as registerSwiper } from 'swiper/element'
 import {
   faAngleDown,
   faAngleUp,
+  faAnglesDown,
+  faAnglesUp,
```

Comment:
```suggestion
  faAngleDown,
  faAnglesDown,
  faAnglesUp,
  faAngleUp,
```
Alphabetized (see comment at top of file)